charts/pvault-server/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: {{ include "pvault-server.capabilities.networkPolicy.apiVersion" . }}
+metadata:
+  name: {{ include "pvault-server.fullname" . }}
+  namespace: {{ include "pvault-server.namespace" . | quote }}
+  labels: {{- include "pvault-server.labels.standard" (dict "customLabels" .Values.commonLabels "context" .) | nindent 4 }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "pvault-server.tplvalues.render" (dict "value" .Values.commonAnnotations "context" .) | nindent 4 }}
+  {{- end }}
+spec:
+  podSelector:
+    matchLabels: {{- include "pvault-server.labels.matchLabels" (dict "customLabels" (include "pvault-server.pod.labels" .) "context" .) | nindent 6 }}
+  policyTypes:
+    - Ingress
+    - Egress
+  {{- if .Values.networkPolicy.allowExternalEgress }}
+  egress:
+    - {}
+  {{- else }}
+  egress:
+    # Allow DNS
+    - ports:
+      - port: 53
+        protocol: TCP
+      - port: 53
+        protocol: UDP
+    {{- if .Values.networkPolicy.extraEgress }}
+    {{- include "pvault-server.tplvalues.render" (dict "value" .Values.networkPolicy.extraEgress "context" .) | nindent 4 }}
+    {{- end }}
+  {{- end }}
+  ingress:
+    # Allow connections to service port
+    - ports:
+        - port: 8123
```

Comment:
.Valuts.service.port ?

Reply:
Or is it just the external port to the service?

Reply:
This is the port for the pod, which is static, should it be configurable ?

Reply:
Updated comment to be more clear

Reply:
nah it's ok np